lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -62,6 +73,21 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                             <span class="action-label">${_("Post on Facebook")}</span>
                         </a>
                       %endif
+                      % if share_settings.get('CERTIFICATE_TWITTER', False):
+                      <% twitter_url = 'https://twitter.com/intent/tweet?text=' + twitter_share_text + '&url='+ share_url%>
+                        <a
+                          data-tooltip="${_('Share on Twitter')}"
+                          class="action action-share-twitter btn btn-overlay btn-small icon-only"
+                          aria-haspopup="true"
```

Comment:
This isn't the right use case for aria-haspopup.  It's a very common misunderstanding.  You can remove it as well as aria-expanded attribute below.